File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Extensibility/1033/Classifier/ClassificationType.vb
Imports System.ComponentModel.Composition
Imports Microsoft.VisualStudio.Text.Classification
Imports Microsoft.VisualStudio.Utilities

Module $safeprojectname$TypeDef

    ''' <summary>
    ''' Defines the "$safeprojectname$" classification type.
    ''' </summary>
    <Export(GetType(ClassificationTypeDefinition))>
    <Name("$safeprojectname$")>
    Private _$safeprojectname$Type As ClassificationTypeDefinition

End Module

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Extensibility/1033/Classifier/ClassificationType.cs
using System.ComponentModel.Composition;
using Microsoft.VisualStudio.Text.Classification;
using Microsoft.VisualStudio.Utilities;

namespace $safeprojectname$
{
    internal static class $safeprojectname$ClassificationDefinition
    {
        /// <summary>
        /// Defines the "$safeprojectname$" classification type.
        /// </summary>
        [Export(typeof(ClassificationTypeDefinition))]
        [Name("$safeprojectname$")]
        internal static ClassificationTypeDefinition $safeprojectname$Type = null;
    }
}
